### ISSUE TITLE
Would it be possible to assign different taxonomies to different templates?

### ISSUE THREAD

**Issue by Protolich** [craftsmancoding/taxonomies#22]
If I have multiple, distinct, types not all the taxonomy categories would make sense for each item.

For example having Cars and Sandwiches. I may have a Fuel Type taxonomy for the cars but this makes no sense to display this for Sandwiches.

**Comment by fireproofsocks** [comment]
Yes, you can assign different taxonomies to different templates by using the taxonomies.skip_templates system setting.  

I'm not sure if it's architecturally a good idea to limit terms in the same way.  In your example, you could have a taxonomy containing terms/categories for vehicle fuels and a separate taxonomy containing terms/categories for food fuels.  Then you could limit which is visible using the taxonomies.skip_templates system setting.  That would accomplish a cleaner UX, but at the cost of a unified "Fuel" taxonomy.
